pkg/controller/utils/common/image.go

```diff
@@ -0,0 +1,37 @@
+package common
+
+import "runtime"
+
+const (
+	// OperandImageOpenJDK envvar name for OpenJDK operand image
+	OperandImageOpenJDK = "RELATED_IMAGE_OPENJDK"
+	// OperandImageOpenJ9 envvar name for OpenJ9 operand image
+	OperandImageOpenJ9 = "RELATED_IMAGE_OPENJ9"
+
+	// DefaultOperandImageOpenJDK default image for OpenJDK stack
+	DefaultOperandImageOpenJDK = "infinispan/server:latest"
+	// DefaultOperandImageOpenJ9 default image for OpenJ9 stack
+	DefaultOperandImageOpenJ9 = "infinispan-for-openj9/server:latest"
+)
+
+// GetDefaultInfinispanImageForCurrentAchitecture returns ...
+func GetDefaultInfinispanImageForCurrentArchitecture() string {
```

Comment:
```suggestion
func GetDefaultInfinispanJavaImage string {
```
and we will reserve `GetDefaultInfinispanNativeImage` for native in the future